PlantPal/PlantPal/View/ReusableView/CustomButton.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  CoustomButton.swift
+//  PlantPal
+//
+//  Created by Marcin Dytko on 09/07/2024.
+//
+
+import SwiftUI
+
+struct CustomButton: View {
+    var title: String
+    var action: () -> Void
```

Comment:
Zamieniłbym to na to:

```suggestion
    private let title: String
    private let action: () -> Void
    
    init(title: String, action: @escaping () -> Void) {
        self.title = title
        self.action = action
    }
```

Reply:
Ogólnie takie podejście daje większa swobodę przy tworzeniu widoku.